addons/web/static/src/views/relational_model.js

```diff
@@ -1183,6 +1318,35 @@ export class Record extends DataPoint {
                 } else {
                     this.data[fieldName] = defaultValues[fieldName];
                 }
+
+                if (field.type === "properties") {
+                    this._applyPropertyDefinitions(fieldName);
+                    for (const property of this.data[fieldName]) {
+                        const fieldPropertyName = `${fieldName}.${property.name}`;
+                        if (!this.activeFields[fieldPropertyName]) {
+                            continue;
+                        }
+                        if (isX2Many(property)) {
+                            const staticList = this._createStaticList(fieldPropertyName);
+                            this._cache[fieldPropertyName] = staticList;
+                            await staticList.setDataRecords(
+                                (property.value || []).map((val) => ({
+                                    id: val[0],
+                                    display_name: val[1] || this.model.env._t("No Access"),
+                                }))
+                            );
+                            this.data[fieldPropertyName] = staticList;
+                        } else if (property.type === "many2one") {
+                            this.data[fieldPropertyName] =
+                                property.value.length && property.value[1] === null
+                                    ? [property.value[0], this.model.env._t("No Access")]
+                                    : property.value;
+                        } else {
+                            this.data[fieldPropertyName] =
+                                property.value === undefined ? false : property.value;
```

Comment:
```suggestion
                                property.value ?? false;
```

if it can be `null` and we want the value to be `null` then don't change